src/render/traverse.js

```diff
@@ -40,13 +57,15 @@ function renderNode (seq, node, context) {
   seq.emit(() => `<${node.type}`);
   seq.emit(() => renderAttrs(node.props, seq));
   seq.emit(() => REACT_ID);
-  seq.emit(() => ">");
+  seq.emit(() => omittedCloseTags.includes(node.type) ? "/>" : ">");
```

Comment:
Isn't it faster to convert this array to `const isOmitted = {area: true, base: true, ...}` and then use `isOmitted[node.type]`?